### PR TITLE
Feat/cli commands

### LINKED ISSUE #92: <cli_name> transform new_data.csv
<cli_name> transform new_data.csv should transform a data set in
the provided format to a data set in the “trip format” and save it to disk (see 1
c)).

### LINKED ISSUE #93: Add cli options listed in project requirements
Create a command-line interface for your machine learning algorithm (Prediction). It
should provide at least three functions:
* <cli_name> train should train an ML model based on data (in the format
 
that data was provided to you) in a defined location and store the model.
* <cli_name> predict new_data.csv should load a previously trained
model and predict journey duration and direction (see above) based on Start information given in new_data.csv (Provided in the same format as other data provided to you). Predictions should be saved to disk.
* <cli_name> transform new_data.csv should transform a data set in the provided format to a data set in the “trip format” and save it to disk (see 1 c)).

### PR DESCRIPTION
**Pull request by MaxStroh**
Added click command and options stated in project tasks.
* nextbike train
* nextbike predict new_data.csv
* nextbike transform new_data.csv
* nextbike descriptive_analysis

Still TODO:
* Move cli_code methods into packages
* Missing help strings & docstrings (TODOs already inserted into code) see #104

### CODE REVIEW

File: nextbike/cli_code.py [hunk 1]
```diff
@@ -0,0 +1,284 @@
+import pandas as pd
+from nextbike import utils, io, datapreparation, prediction, testing, visualization
+
+
+def cleaning(filename="nuremberg.csv"):
```
Comment (weisske) [inline nextbike/cli_code.py:5]: move all of these functions to packages.

Comment (weisske) [reply nextbike/cli_code.py:5]: #116

File: nextbike/cli.py [hunk 2]
```diff
@@ -31,323 +26,115 @@ def main(test, clean, viso, train, pred, traingeo, predgeo, weather):
...
+    start_time_desc_analysis = datetime.now().replace(microsecond=0)
+    start_time_step = start_time_desc_analysis
+    if clean:
+        print("START CLEAN")
+        cli_code.cleaning(filename)
+        start_time_step = utils.print_time_for_step("STEP CLEAN", start_time_step)
+    print("START VISUALIZATION")
+    cli_code.visualize()
+    utils.print_time_for_step("STEP VISUALIZATION", start_time_step)
+    utils.print_time_for_step("COMMAND DESCRIPTIVE ANALYSIS", start_time_desc_analysis)
```
Comment (weisske) [inline nextbike/cli.py:136]: Very nice option!

**Review by weisske — APPROVED** [review]
Do it!

**Review by weisske — APPROVED** [review]

**Review by mulko93 — APPROVED** [review]
looks great and can be merged

**Review by Speedfan94 — APPROVED** [review]
Approved

**Pull request merged**